pkg/controller/windowsmachineconfig/windows/windows.go

```diff
@@ -111,6 +115,8 @@ func New(vm types.WindowsVM, workerIgnitionEndpoint string, signer ssh.Signer) (
 		nil
 }
 
+// Interface methods
```

Comment:
Unnecessary?

Reply:
No. I want to group these methods together. I have similar comments for other method types.